packages/fxa-payments-server/src/lib/formats.ts

```diff
@@ -44,7 +44,7 @@ export function getLocalizedCurrencyString(
   const decimal = (amountInCents || 0) / 100;
   const options = { ...baseCurrencyOptions, currency };
 
-  return new Intl.NumberFormat('en-US', options).format(decimal);
+  return new Intl.NumberFormat('en', options).format(decimal);
```

Comment:
@StaberindeZA This is definitely one I'd want SubPlat to check on. 🤔

Reply:
Looks good to me. We only use it for fallback text, and it seems to still work as expected. Thank you!